File: bddtests/src/main/kotlin/mocking/vision/models/EligibleRepeats.kt
package mocking.vision.models

import javax.xml.bind.annotation.XmlElement
import javax.xml.bind.annotation.XmlRootElement

@XmlRootElement(name = "eligibleRepeats")
data class EligibleRepeats(

        @XmlElement(namespace= "urn:vision", name = "settings")
        var settings: Settings = Settings(true),

        @XmlElement(namespace= "urn:vision", name = "repeat")
        var repeat: MutableList<RepeatCourse> = mutableListOf()

)

File: backendworker/NHSOnline.Backend.GpSystems/Suppliers/Vision/Models/Courses/EligibleRepeats.cs
using System.Collections.Generic;
using System.Xml.Serialization;

namespace NHSOnline.Backend.GpSystems.Suppliers.Vision.Models.Courses
{
    [XmlRoot(ElementName = "eligibleRepeats", Namespace = "urn:vision")]
    public class EligibleRepeats
    {
        [XmlElement(ElementName = "settings", Namespace = "urn:vision")]
        public CourseSettings Settings { get; set; }

        [XmlElement(ElementName = "repeat", Namespace = "urn:vision")]
        public List<Repeat> Repeats { get; set; } = new List<Repeat>();
    }
}